lib/appsignal/phoenix/instrumenter.ex

```diff
@@ -18,68 +18,49 @@ if Appsignal.phoenix? do
     Note: Channels (`phoenix_channel_join` hook) are currently not
     supported.
 
-    ## Custom instrumentation events
-
-    You might be using your endpoint's `instrument/4` macro to create
-    custom instrumentation. If you want those events to become part of
-    the AppSignal timeline as well, you need to create a custom
-    instrumenter module with the help of
-    Appsignal.Phoenix.InstrumenterDSL:
-
-    ```
-    defmodule PhoenixApp.CustomInstrumenter do
-
-      import Appsignal.Phoenix.InstrumenterDSL
-      instrumenter :phoenix_controller_call
-      instrumenter :phoenix_controller_render
-      instrumenter :custom_event
-      instrumenter :another_custom_event
-    end
-    ```
-
-    And then, use that instead of the AppSignal instrumenter in your `config.exs`:
-
-    ```
-    config :phoenix_app, PhoenixApp.Endpoint,
-      instrumenters: [PhoenixApp.CustomInstrumenter]
-    ```
-
     See the [Phoenix integration guide](phoenix.html) for information on
     how to instrument other aspects of Phoenix.
     """
+    @transaction Application.get_env(:appsignal, :appsignal_transaction, Appsignal.Transaction)
 
-    alias Appsignal.Transaction
+    @doc false
+    def phoenix_controller_call(:start, _, args), do: start_event(args)
 
-    require Logger
+    @doc false
+    def phoenix_controller_call(:stop, _diff, {%Appsignal.Transaction{} = transaction, %{conn: conn} = args}) do
+      finish_event(transaction, "phoenix_controller_call", args)
```

Comment:
We discussed changing this name so that it doesn't get put in the "other" group. Can we change this to the event name `controller_call.phoenix`?